### DISCUSSION TITLE
adding AI to SearXNG

### DISCUSSION THREAD

**Discussion by MrNobody334** [searxng/searxng#2418] [Ideas]
Is it possible to add AI for enhance the search results ?  
I think there is a issue of image searching. In this case I can't use image searching. It appears nothing in the relevant tab.

![image](https://github.com/searxng/searxng/assets/91918729/097d8734-9327-4515-8351-5a6d8e74e0f3)

**Comment by return42** [comment]
> Is it possible to add AI for enhance the search results ?

discussed here https://github.com/searxng/searxng/issues/2273#issuecomment-1479178449

> I think there is a issue of image searching. 

Different topic / for the future please do not mix different topics in one thread ... but to answer your question and finalize see: 

- https://github.com/searxng/searxng/issues/2399

**Reply by MrNobody334** [reply]
Sorry for the mistake.. 
and also sorry for asking that I cannot understand the solution for image thumbnail problem. Can you explain it if you don't mind?
